build/builder.sh

```diff
@@ -5,20 +5,9 @@ set -eu
 image="cockroachdb/builder"
 
 function init() {
-  docker build --tag="${image}" - <<EOF
-FROM golang:1.5
-
-RUN curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
- apt-get dist-upgrade -y && \
- apt-get install --no-install-recommends --auto-remove -y git build-essential file nodejs && \
- apt-get clean autoclean && \
- apt-get autoremove -y && \
- rm -rf /tmp/*
-RUN go get golang.org/x/tools/cmd/vet
-ENV SKIP_BOOTSTRAP=1
-
-CMD ["/bin/bash"]
-EOF
+  cp "$(dirname $0)"/../ui/{bower,npm-shrinkwrap,package,tsd}.json "$(dirname $0)"/../ui/Makefile "$(dirname $0)"
```

Comment:
are we going to frequently update these? Just asking since any change here requires a manual tag change in `circle-deps` (which isn't nicely automated b/c of likely caching issues).